### PR TITLE
Swap the two setfacl commands

### PR DESCRIPTION
**Pull request by ghost**
The description was not suitable for the given setfacl commands.

### CODE REVIEW

**Comment by xabbuh** [comment]
We recently swapped the calls for good reasons in #7987. I think we should then update the description instead if it doesn't match the example.

File: setup/file_permissions.rst [hunk 1]
```diff
@@ -55,8 +55,7 @@ following script to determine your web server user and grant the needed permissi
 
 .. note::
 
-    The first ``setfacl`` command sets permissions on the existing files and
-    folders, while the second one sets permissions for future files and folders.
+    The first ``setfacl`` command sets permissions for future files and folder, while the second one sets permissions on the existing files and folders.
```
Comment (wouterj) [inline setup/file_permissions.rst]: If you add a line break after the first word crossing the 72th character, this PR would be perfect! :) (if you don't have the time, we can do it during merging).

**Review by wouterj — APPROVED** [review]
:+1:

**Comment by xabbuh** [comment]
Thank you @zephyr-thomas.

**Pull request closed without merging**